src/style.css

```diff
@@ -0,0 +1,198 @@
+@font-face {
+  font-family: Roboto;
+  src: url(./fonts/Roboto-Regular.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(./fonts/Roboto-Medium.ttf);
+  font-style: normal;
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(./fonts/Roboto-Bold.ttf);
+  font-style: normal;
+  font-weight: 700;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+ul,
+ol {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+li,
+button {
+  margin: 0;
+  padding: 0;
+}
+
+button {
+  background: none;
+  border: none;
+  outline: none;
+  cursor: pointer;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+body {
+  margin: 0 auto;
+  font-family: Roboto, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+:root {
+  --main-accent: #060b35;
+  --secondary-color: #616070;
+  --blue-accent: #00acdc;
+  --border-color: #f3f3f3;
+  --white: #fff;
+}
+
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  border: 0;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0,0,0,0);
+  white-space: nowrap;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: "Roboto", sans-serif;
+  background: var(--white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__heading {
+  margin-bottom: 5px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 150%;
+  color: var(--main-accent);
+}
+
+.card__details {
+  margin-bottom: 18px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 116%;
+  color: var(--secondary-color);
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 23px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star.svg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 13px 12px;
+}
+
+.stars--4 > .stars__star:not(:last-child) {
+  margin-right: 4px;
+  background-image: url("images/star-active.svg");
+}
+
+.card__rewiews {
+  font-size: 10px;
+  font-weight: 400;
+  font-style: normal;
+  color: var(--main-accent);
+}
+
+.card__price-section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-text {
+  font-size: 12px;
+  font-weight: 400;
+  font-style: normal;
+  color: var(--secondary-color);
+}
+
+.card__price-number {
+  font-size: 16px;
+  font-weight: 700;
+  font-style: normal;
+  color: var(--main-accent);
+}
+
+.card__btn {
+  display: block;
+  padding: 12px 0;
+  text-align: center;
+}
+
+.blue-link {
```

Comment:
don't create separate BEM block blue-link, set additional modifiers to the card element